src/test/java/com/epam/tc/hw7/test/TestBase.java

```diff
@@ -0,0 +1,35 @@
+package com.epam.tc.hw7.test;
+
+import static com.epam.jdi.light.settings.WebSettings.logger;
+import static com.epam.tc.hw7.states.States.shouldBeLoggedOut;
+
+import com.epam.jdi.light.driver.WebDriverUtils;
+import com.epam.jdi.light.elements.init.PageFactory;
+import com.epam.tc.hw7.SiteJdi;
+import org.assertj.core.api.SoftAssertions;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeSuite;
+
+public class TestBase {
+
+    public SoftAssertions softAssertions;
+
+    @BeforeSuite
+    void beforeSuite() {
+        softAssertions = new SoftAssertions();
+        PageFactory.initSite(SiteJdi.class);
+        logger.info("Run Tests");
+    }
+
+    @BeforeMethod
+    void beforeMethod() {
+        shouldBeLoggedOut();
```

Comment:
Why we should be logged out?

Reply:
We must logged out, because in the steps we must log in each time, and for this we must at the end of each method logged out

Reply:
We have an opposite requirements for test: we should be logged in.
Performance is a very important part of AT

Reply:
Fix. I transfer it to after method, because for next data set we need log in again, so in postconditions we logged out.

Reply:
And it's still have the same wrong logic

Reply:
In step one we login on site, so for this in the previous step, we must to logged out